AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -785,6 +785,33 @@ - (NSMutableDictionary *)getReportingParametersForUpdatedRelease:(NSString *)upd
   return reportingParameters;
 }
 
+- (void)changeDistributionGroupIdAfterAppUpdateIfNeeded {
+  NSString *lastDownloadedReleaseHashes = [MS_USER_DEFAULTS objectForKey:kMSDownloadedReleaseHashKey];
+  NSString *lastDownloadedDistributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDownloadedDistributionGroupIdKey];
+  if (lastDownloadedDistributionGroupId == nil || ![self isCurrentReleaseWasUpdated:lastDownloadedReleaseHashes]) {
+    return;
+  }
+  NSString *currentDistributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
+  if (currentDistributionGroupId != nil && [lastDownloadedDistributionGroupId isEqualToString:currentDistributionGroupId]) {
```

Comment:
Please surround chained conditions with `()`. I know this seams like a detail but as we have more people working on the code base, I'd like us to be more careful about clean code and formatting, etc.

Reply:
Fixed. Thank you.